src/main/java/uk/gov/companieshouse/api/strikeoffobjections/controller/ObjectionController.java

```diff
@@ -146,7 +149,15 @@ public ObjectionController(PluggableResponseEntityFactory responseEntityFactory,
         );
 
         try {
-            Objection objection = objectionService.createObjection(requestId, companyNumber, ericUserId, ericUserDetails);
+            ObjectionUserDetails userDetails = objectionService.buildUserDetails(
```

Comment:
Kind of related to @ICoxCH 's comment below, but why not just pass the 'objectionCreate' object into the existing 'createObjection()' method and let it do what it needs to (in a private service method) to build an object to save in Mongo?

Reply:
It's undesirable because we end up with a method with a parameter list of five.

Reply:
Fair enough. Maybe just have 'buildUserDetails()' as a private helper method in this controller class then?

Reply:
will live with the extra parameter.